src/schema/type/DefinedTerm.graphql

```diff
@@ -0,0 +1,70 @@
+"https://pending.schema.org/DefinedTerm"
+type DefinedTerm implements ThingInterface {
+    ### Metadata properties ###
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID
+    "http://purl.org/dc/elements/1.1/contributor,https://schema.org/contributor"
+    contributor: String
+    "http://purl.org/dc/elements/1.1/coverage"
+    coverage: String
+    "http://purl.org/dc/elements/1.1/creator"
+    creator: String!
+    "http://purl.org/dc/elements/1.1/date"
+    date: _Neo4jDate
+    "http://purl.org/dc/elements/1.1/description,https://schema.org/description"
+    description: String
+    "https://schema.org/disambiguatingDescription"
+    disambiguatingDescription: String
+    "http://purl.org/dc/elements/1.1/format"
+    format: String
+    "http://purl.org/dc/elements/1.1/language"
+    language: AvailableLanguage
+    "http://purl.org/dc/elements/1.1/publisher"
+    publisher: String
+    "http://purl.org/dc/elements/1.1/relation"
+    relation: String
+    "http://purl.org/dc/elements/1.1/rights"
+    rights: String
+    "http://purl.org/dc/elements/1.1/source"
+    source: String
+    "http://purl.org/dc/elements/1.1/subject"
+    subject: String
+    "http://purl.org/dc/elements/1.1/title"
+    title: String
+    "http://purl.org/dc/elements/1.1/type,https://www.w3.org/2000/01/rdf-schema#type"
+    type: String
+    "http://purl.org/dc/elements/1.1/date"
+    created: String
+    "http://purl.org/dc/elements/1.1/date"
+    updated: String
+    #################################
+    ### ThingInterface properties ###
+    "https://schema.org/additionalType"
+    additionalType: [String]
```

Comment:
By convention, @musicog and I decided that this field will be "https://vocab.trompamusic.eu/vocab#TagCollectionElement" for items which refer to a set of tags used in an annotation. We'll make sure to add this to the documentation for annotations